context/app/static/js/shared-styles/cards/SelectableCard/SelectableCard.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { ChangeEvent } from 'react';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Chip from '@mui/material/Chip';
+import Typography, { TypographyProps } from '@mui/material/Typography';
+import Stack from '@mui/material/Stack';
+import Checkbox from '@mui/material/Checkbox';
+import { styled } from '@mui/material/styles';
+import { SystemStyleObject } from '@mui/system';
+
+interface SelectableCardProps {
+  title: string;
+  description: string;
+  tags?: string[];
+  isSelected: boolean;
+  selectItem: (e: ChangeEvent<HTMLInputElement>) => void;
+  cardKey: string;
+  sx?: SystemStyleObject | SystemStyleObject[];
+}
+
+interface SelectableCardTextProps extends TypographyProps {
+  $colorVariant: 'primaryContainer' | 'secondaryContainer';
+}
+
+const SelectableCardText = styled(Typography)<SelectableCardTextProps>(({ theme, $colorVariant }) => ({
+  color: theme.palette[$colorVariant].contrastText,
+}));
+
+function SelectableCard({
+  title,
+  description,
+  tags = [],
+  isSelected,
+  selectItem,
+  cardKey,
+  sx = [],
+}: SelectableCardProps) {
+  const colorVariant = isSelected ? 'primaryContainer' : 'secondaryContainer';
+  return (
+    <Card
+      sx={[
+        {
+          minWidth: 275,
+          backgroundColor: `${colorVariant}.main`,
+        },
+        ...(Array.isArray(sx) ? sx : [sx]),
+      ]}
+    >
+      <CardContent component={Stack} direction="column" sx={{ height: '100%' }}>
+        <Stack direction="row" sx={{ justifyContent: 'space-between', alignItems: 'center' }}>
```

Comment:
`Stack` has `justifyContent` and `alignItems` as native props: https://mui.com/material-ui/react-stack/#interactive-demo